xml/System.Net.Security/TlsCipherSuite.xml

```diff
@@ -59,7 +68,7 @@
       </ReturnValue>
       <MemberValue>4868</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Represents the TLS_AES_128_CCM_SHA256 cipher suite.</summary>
```

Comment:
Although these descriptions are OK as it, it would be nice to shorten them by removing "Represents the ".

Reply:
@Jozkee used this same description in a template in dotnet/corefx, so modifying these descriptions will mean modifying the ones in the template too.

If it's not blocking, I'd say we keep the current description as it is now.

Reply:
That's fine.